requirements.txt

```diff
@@ -1,3 +1,3 @@
-flask
-requests
-logicmonitor_sdk
\ No newline at end of file
+flask==2.0.3
+requests==2.27.1
```

Comment:
I cringe at pinning `requests`, but you can do it if you'd like.

`requests` is pinned by StackStorm itself. So, this will probably result in conflicts at some point (most likely when we stop using the deprecated legacy resolver in pip - No ETA for when that will happen).
https://github.com/StackStorm/st2/blob/master/requirements.txt#L65

Reply:
Addressed in recent commit